bcda/main.go

```diff
@@ -59,6 +60,20 @@ func init() {
 	log.Info(fmt.Sprintf(`Auth is made possible by %T`, auth.GetProvider()))
 }
 
+func createAPIDirs() {
+	payload := os.Getenv("FHIR_PAYLOAD_DIR")
+	err := os.MkdirAll(payload, os.ModePerm)
```

Comment:
`os.ModePerm` = 777, right? Do we want these locked down a bit more, maybe 644?

Reply:
changed to 0750 and it works but willing to try something else if people have ideas.

Reply:
changed to 0744 after discussing with @knollfear. Good stuff!